third_party/amd/lib/TritonAMDGPUToLLVM/Utility.h

```diff
@@ -22,6 +22,12 @@ Value shflIdxSync(Location loc, ConversionPatternRewriter &rewriter, Value val,
                   int i);
 Value shflIdxSync(Location loc, ConversionPatternRewriter &rewriter, Value val,
                   Value i);
+Value storeShared(ConversionPatternRewriter &rewriter, Location loc, Value ptr,
+                  Value val, Value pred);
+Value loadShared(ConversionPatternRewriter &rewriter, Location loc, Value ptr,
+                 Type elemTy, Value pred, const std::string& val);
```

Comment:
It's not obvious (to me, anyway) how `val` works in this function call.  Please add comments and change the variable names.